audio/voices.yaml

```diff
@@ -1,28 +1,26 @@
-AlliedEngineerVoice:
-	DefaultVariant: .wav
-	Voices:
-		Select: ienasea, ienaseb, ienasec, ienased
-		Move: ienamoa, ienamob, ienamoc
-		Action: ienaata, ienaatb, ienaatc
-		Feedback: ienafea, ienafeb, ienafec
-		Die: ienadia, ienadib, ienadic, ienadid
+EngineerVoice:
+	DefaultVariant: .wav
+	Prefixes:
```

Comment:
Missing files are hard to find in this system already. Sound.log failure is hard to notice already and fixing this would mean even less chances to debug/notice missing soundfiles.